.github/workflows/rust.yml

```diff
@@ -17,6 +17,8 @@ jobs:
         # meantime, the build only tests Windows Server 2019.
         # See https://github.com/amazon-ion/ion-rust/issues/353
         os: [ubuntu-latest, windows-2019, macos-latest]
+        # build and test for different and interesting crate features
+        features: ['default', 'all', 'ion-hash', 'experimental', 'ion-hash,experimental']
```

Comment:
Should we move `ion-hash` under the `experimental` umbrella?

Reply:
🤷‍♀️ - Do we consider it experimental?  Cut #534 to deal with this.